aws_lambda_builders/workflows/ruby_bundler/bundler.py

```diff
@@ -65,13 +66,19 @@ def run(self, args, cwd=None):
                 if self.osutils.directory_exists(check_dir):
                     self.osutils.remove_directory(check_dir)
             else:
-                # Bundler has relevant information in stdout, not stderr.
+                # Bundler can contain information in both stdout and stderr so we check and log both
+                err_str = err.decode("utf8").strip()
+                out_str = out.decode("utf8").strip()
                 if out and err:
```

Comment:
We are calling `decode` method before if statement, wouldn't this cause issues if `err` or `out` wouldn't exist?

If `out` and `err` exists all the time, then we might need to change `if` statement to `if err_str and err_str` so that it will be more clear.

Reply:
Then I think we should update the if statements to check `err_str` and `out_str` to act on them.